### PR TITLE
Add processing of uploaded 23andMe data

### PR DESCRIPTION
**Pull request by madprime**

### CODE REVIEW

**Comment by beaugunderson** [comment]
@madprime can we pass a full temporary URL to data-processing instead of the bucket and key?

**Comment by madprime** [comment]
@beaugunderson yeah, I can update this, sounds fine

**Comment by madprime** [comment]
Also need to make sure it's correctly parsing the task parameters as defined here: https://github.com/PersonalGenomesOrg/open-humans/blob/master/activities/twenty_three_and_me/models.py#L57

File: data_retrieval/go_viral.py [hunk 1]
```diff
@@ -48,16 +49,16 @@ def create_go_viral_ohdataset(access_token, go_viral_id,
...
     'filedir') must be specified.
     """
-    identifier = go_viral_id.replace('simplelogin:', 'individual-')
-    filename = format_filename('go-viral', identifier, 'sickness-reports')
+    filename = format_filename(source='go-viral',
+                               data_type='sickness-and-viral-profiling')
 
     source = OHDataSource(name='GoViral Integration API',
-                          url='http://www.goviralstudy.com/')
+                          url='http://www.goviralstudy.com/',
```
Comment (beaugunderson) [inline data_retrieval/go_viral.py]: @madprime should also change to https://?

**Comment by beaugunderson** [comment]
@madprime looks good, one comment inline

also, I hadn't thought to include pip-tools in the requirements but I think it makes sense :)

**Pull request merged**